### DISCUSSION TITLE
only one tibber feed

### DISCUSSION THREAD

**Discussion by Ossiostborn** [bisand/node-red-contrib-tibber-api#98] [Q&A]
not unnormal. But I have to read livedata from more than one home Id. What is best practice for this use case?

**Comment by bisand** [comment]
Have you tried to provide two different TibberFeed nodes with different home ID's?
![image](https://github.com/user-attachments/assets/3d055ea0-92cf-4665-a59a-bd4d5b6da374)

I do not have two homes so I cannot test for myself.
